docs/modules/advanced_speed_servo_info.rst

```diff
@@ -14,6 +14,9 @@ Supported Features
         * :ref:`manual_timeout`
         * :ref:`manual_zero_spin`
         * :ref:`controlling_ifci`
+
+Gen 2 Modules Only
```

Comment:
Not really a big deal, but this is slightly confusing on the 40 and 60 pages since they only have gen 2 versions, unlike the 81. Not sure if there's anyway to have them show up a bit differently, maybe with this just in the feature list without the gen 2 only thing, that isn't deeply painful. If its super annoying, you don't need to worry about it

Reply:
@vertiq-jordan On this one, what I was curious about was if we could have the 81 page have the gen 2 thing, since its relevant there, and the 40s and 60s not have the gen 2 thing, since its not relevant there. Seems like now none of them say the gen 2 thing. Is there a way to split the difference on these, or is that not possible?

Reply:
Ah I see. I'll make it happen.